consumption-interface/Release_Notes_1_0_0.md

```diff
@@ -0,0 +1,15 @@
+Known issues and limitations of the Consumption Interface Supervisor Service
```

Comment:
missing "Local" before Consumption Interface Supervisor Service.

Reply:
This also encompasses the SSO support. The name is correct.